app/models/license.rb

```diff
@@ -61,4 +75,7 @@ def redo_project_licenses(project_licenses)
       pl.edits.undone_by_hamster.each { |edit| edit.redo!(Account.hamster) }
     end
   end
+
+  def add_required_license_permission(licence_permission)
+  end
```

Comment:
This looks like a thought in process that was never completed and can be removed.